privacyidea/lib/token.py

```diff
@@ -341,6 +347,25 @@ def get_tokens(tokentype=None, realm=None, assigned=None, user=None,
     # Decide, what we are supposed to return
     if count is True:
         ret = sql_query.count()
+    elif psize:
+        count = sql_query.count()
```

Comment:
I think we could avoid one SQL query here by using ``pagination.total``.

Reply:
done